docs/bundles/SyliusGridBundle/configuration.rst

```diff
@@ -5,34 +5,30 @@ Configuration Reference
 
     sylius_grid:
         grids:
-            app_user: # Your grid name.
-                driver: doctrine/orm # Data source driver.
-                resource: app.user # Resource name.
+            app_user: # Your grid name
+                driver: doctrine/orm # Data source driver
+                resource: app.user # Resource name
```

Comment:
Should be:

```
driver: 
    name: doctrine/orm
    options:
        class: AppBundle\Entity\User
```

See: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminBundle/Resources/config/grids/channel.yml#L4